gramps/gui/views/treemodels/peoplemodel.py

```diff
@@ -265,7 +265,9 @@ def _get_birth_data(self, data, sort_mode):
             event = self.db.get_event_from_handle(er.ref)
             etype = event.get_type()
             date_str = get_date(event)
-            if (etype in [EventType.BAPTISM, EventType.CHRISTEN]
+            if (etype in [EventType.STILLBIRTH,
```

Comment:
Use `etype.is_birth_fallback()` instead.

Reply:
The same issue appeared at line 367 so also made that change.

Fixed in [15a0085](https://github.com/gramps-project/gramps/pull/1148/commits/15a0085b6316bb277400a9133eabf32b47be2558)